utils/utils.go

```diff
@@ -209,3 +209,12 @@ func ExtractDefaultLabels(instanceId string, details brokerapi.ProvisionDetails)
 
 	return sanitized
 }
+
+func LineErrorFormatter(es []error) string {
```

Comment:
This should probably have a godoc

Reply:
Renamed this to `SingleLineErrorFormatter` and added godoc.